packages/node_modules/@webex/plugin-meetings/src/locus-info/infoUtils.js

```diff
@@ -67,10 +67,7 @@ InfoUtils.getInfos = (oldInfo, newInfo) => {
   return {
     previous,
     current,
-    updates: {
-      isLocked: InfoUtils.isLocked(current),
```

Comment:
this (and next line) never worked, because `isLocked()`/`isUnlocked()` take policy as argument so it should be `current.policy` and not `current`. I've decided to just remove this code, because we seem to have code that has more specific logic for setting the updates a few lines above.